### PR TITLE
add Flask API on Azure Container Apps and Todo Nodejs Mongo AKS template

### LINKED ISSUE #97: [New azd-template] Flask API on Azure Container Apps
## New azd-template Submission
Fill in the sections below to populate a new gallery card for your template. **All fields are mandatory unless explicitly identified as optional**

## 1. Description

A simple JSON API using the Python Flask framework, designed for deployment on Azure Container Apps.

## 2. Author Information
**Name:** Pamela Fox

**Link:** https://github.com/pamelafox

## 3. Template Repo
_The GitHub repo containing the template being contributed-- must be a valid, complete template_

https://github.com/pamelafox/simple-flask-api-container

## 4. Link to PR where you made your repo `azd` compatible
_To allow us to provide feedback on your template, please include a link to the PR where you added `azd` compatibility_

Sorry, no distinct PR. Check the .github, .devcontainer, and infra folders.

## 5. Tags
**Standard:** _One or more tags representing the template. Look at the currently available tags on the [Gallery page](https://azure.github.io/awesome-azd/). Provide **at least 1 tag** for programming language used and **at least 1 tag for Azure services integrated**. Also tag the **IaC provider** (Bicep or Terraform)_

Python, Bicep, Azure Container Apps, Flask

**Additional:** _Don't see a relevant tag for your template? Suggest tags to be added and used for your template. These could be new languages, services, or application domains_.

## 6. Architecture Diagram or Application Screenshot
_Used as display image for gallery card-- should include all services and their connections. [Example](https://github.com/Azure-Samples/todo-csharp-sql/blob/main/assets/resources.png)_

Architecture diagram

## 7. Related Issues (optional)
_Is this a template that was requested in a separate issue? If so, please tag the related issue here._
---
## Next Steps
Once submitted, the issue will be reviewed - we plan to do reviews on a rolling basis at regular intervals. The process will include verifying all information required for the template gallery is provided and the template works (i.e., successfully deploys to Azure with `azd up`). 
 * If we have questions or enhancements, we will add comments in issue thread (**issue stays open**)
 * If the contribution is approved, we'll update the gallery directly (**issue will then be closed**)

### PR DESCRIPTION
**Pull request by hemarina**
fix #97 
Add https://github.com/azure-samples/todo-nodejs-mongo-aks

### CODE REVIEW

File: website/src/data/users.tsx [hunk 1]
```diff
@@ -408,6 +408,16 @@ const Users: User[] = [
...
   },
+  {
+    title: 'Flask API on Azure Container Apps',
+    description:
+      'A simple JSON API using the Python Flask framework, designed for deployment on Azure Container Apps.',
+      preview: require('./images/simple-flask-api-container.png'),
+    website: 'https://github.com/pamelafox',
+    author: 'Pamela Fox',
+    source: 'https://github.com/pamelafox/simple-flask-api-container',
+    tags: ['python', 'bicep', 'azurecontainerapps', 'flask'],
```
Comment (savannahostrowski) [inline website/src/data/users.tsx]: I just realized that any tag that's "azurecontainerapps" should actually be "aca". We should update this and the Dapr samples we merged yesterday.

**Review by savannahostrowski — APPROVED** [review]

File: website/src/data/users.tsx [hunk 2]
```diff
@@ -314,7 +324,7 @@ const Users: User[] = [
       website: 'https://github.com/Azure-Samples',
       author: 'Azure Content Team',
     source: 'https://github.com/Azure-Samples/svc-invoke-dapr-csharp',
-    tags: ['dapr','csharp','azurecontainerapps']
```
Comment (jongio) [inline website/src/data/users.tsx:317]: Why change to aca? Not sure that will mean anything to people filtering...

Comment (savannahostrowski) [reply website/src/data/users.tsx:317]: This doesn't change the publicly visible tag. The tag in the site should be "Azure Container Apps", this is just what's currently being used on the backend.

Comment (hemarina) [reply website/src/data/users.tsx:317]: image
Yep. We have duplicate tags, and we are using one of them, which is `aca`. Users will see the full name on frontend.

**Pull request merged**